### PR TITLE
fix(package.json): Update dependencies

### PR DESCRIPTION
**Pull request by fhemberger**
The current version of `flaschenpost` contains some dependencies with known vulnerabilities:

```
found 29 vulnerabilities (8 low, 14 moderate, 6 high, 1 critical) in 5653 scanned packages
  run `npm audit fix` to fix 14 of them.
  15 vulnerabilities require semver-major dependency updates.
```

I updated **all** the dependencies to the latest version and fixed the pre/post handler for the tests.

Tested on `node@10.x.x`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,31 +38,31 @@
...
-    "express": "4.16.2",
-    "fs-extra": "5.0.0",
-    "morgan": "1.9.0",
+    "assertthat": "2.0.0",
+    "express": "4.16.4",
+    "fs-extra": "7.0.1",
+    "morgan": "1.9.1",
     "nodeenv": "1.0.0",
-    "roboter": "0.16.0",
+    "roboter": "3.0.3",
```
Comment (goloroden) [inline package.json]: roboter >= 1.0 is not backwards-compatible to the old roboter, so you should also delete the `roboter.js` file, and remove the dependency to `roboter-server`. Please refer to https://github.com/thenativeweb/roboter/blob/master/UPGRADING-0.x-TO-1.0.md for details (and please note that you also have to change `lib` to `src`).

Please also adjust the call mentioned in `README.md` to `npx roboter` to run the build.

In addition to that – could you please also add yourself as a collaborator to the `package.json` file?

File: test/units/post.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
 'use strict';
 
-const fs = require('fs'),
+const fs = require('fs').promises,
```
Comment (goloroden) [inline test/units/post.js]: This is not backwards-compatible to old Node.js versions, could you please use the [util.promisify](https://www.npmjs.com/package/util.promisify) module for that?

**Comment by fhemberger** [comment]
Ok, this should do the trick. PTAL.

File: test/units/post.js [hunk 3]
```diff
@@ -1,12 +1,15 @@
 'use strict';
 
-const fs = require('fs'),
-      path = require('path');
+const util = require('util');
```
Comment (goloroden) [inline test/units/post.js]: Just one more thing: Could you please use the [util.promisify](https://www.npmjs.com/package/util.promisify) module?

**Comment by fhemberger** [comment]
Done & done.

**Pull request merged**